pymemcache/client/base.py

```diff
@@ -739,6 +744,22 @@ def _raise_errors(self, line, name):
             error = line[line.find(b' ') + 1:]
             raise MemcacheServerError(error)
 
+    def _key_helper(self, key, remapped_keys, prefixed_keys):
```

Comment:
While we do need to eventually override this method's logic, by itself it is not particularly useful. It may be more useful to extract the logic under `elif line.startswith(b'VALUE'):` to a separate method. Can you elaborate on the issues you ran into when you tried to do that?

From a cursory glance seems like you can pass in `remapped_keys`, `prefixed_keys`, and `line` as inputs and keep most of the logic intact.